contracts/NodeDelegator.sol

```diff
@@ -29,6 +29,12 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
 
     uint256 internal constant DUST_AMOUNT = 10;
 
+    struct Validator {
```

Comment:
nit: maybe ValidatorDepositData would be a better name for this to be more aligned with P2P terminology. The signature and depositDataRoot only pertain to depositing ETH. The pubkey is a permanent property of the validator

Reply:
I've renamed to `ValidatorStakeData`